things/mocks/channels.go

```diff
@@ -196,17 +201,21 @@ func (crm *channelRepositoryMock) Remove(_ context.Context, owner, id string) er
 
 func (crm *channelRepositoryMock) Connect(_ context.Context, owner string, chIDs, thIDs []string) error {
 	for _, chID := range chIDs {
-		ch, err := crm.RetrieveByID(context.Background(), owner, chID)
+		ch, err := crm.RetrieveByID(context.Background(), chID)
 		if err != nil {
 			return err
 		}
 
 		for _, thID := range thIDs {
-			th, err := crm.things.RetrieveByID(context.Background(), owner, thID)
+			th, err := crm.things.RetrieveByID(context.Background(), thID)
 			if err != nil {
 				return err
 			}
 
+			if owner != ch.Owner {
+				return errors.ErrNotFound
+			}
```

Comment:
Remove this please, eventually change the test if need but this should be verified in the Service layer. Connect justcreates a relation between a thingID to a channelID.